src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -247,15 +267,23 @@ public enum RisingRivals implements LogicCardInfo {
           resistance P, MINUS20
           pokeBody "Eerie Aura", {
             text "Put 1 damage counter on each of your opponent’s Pokémon that remains Asleep between turns."
-            delayedA {
+            before BEGIN_TURN, {
+              if(my.active.isSPC(ASLEEP)){
+                directDamage 10, self, SRC_ABILITY
```

Comment:
This is also not an Ability. Gen4 is where we'll have to start paying attention to the fact that PokemonPowers and Abilities are completely different and have completely different rulings.